public class FunctionUse4404 {

    public void funcUse() {
		org.apache.hadoop.fs.PathIOException pathioexception = new org.apache.hadoop.fs.PathIOException();
		pathioexception.withFullyQualifiedPath(null);
    }
}